### DISCUSSION TITLE
co-pilot in mscode bug clears the chat text box and you lose carefully typed text and have to start again

### DISCUSSION THREAD

**Discussion by DeveloperChris** [community/community#81209] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Using co-pilot (paid not free) in vscode and after writing a (sometimes long prompt). if you scroll up through the prompt and then scroll back down and press the down arrow once too many times the entire prompt is removed and is unrecoverable. It means you have to retype the entire prompt again.

Update: Very annoyingly I just found if you press the up arrow to go to the beginning of your prompt and again go too far it also clears the text box.

**Comment by ericslandry** [comment]
https://github.com/microsoft/vscode/issues/201269
